api/db/repositories/productRepository.js

```diff
@@ -51,39 +64,42 @@ exports.deleteProduct = (productId) => {
             })
             .then((removedRows) => {
                 affectedRows += removedRows;
-                resolve(affectedRows);
+                return affectedRows;
```

Comment:
if I think about it I suppose you can remove `affectedRows` variable and just return `removedRows` ?